RetailCoder.VBE/Refactorings/ImplementInterface/ImplementInterfaceRefactoring.cs

```diff
@@ -70,10 +70,11 @@ private void ImplementMissingMembers()
             var implementedMembers = GetImplementedMembers();
 
             var nonImplementedMembers =
-                interfaceMembers.Where(
-                    d =>
-                        !implementedMembers.Select(s => s.IdentifierName)
-                            .Contains(_targetInterface.ComponentName + "_" + d.IdentifierName)).ToList();
+                interfaceMembers.Where(d => !implementedMembers.Select(s => s.IdentifierName)
+                                    .Contains(_targetInterface.ComponentName + "_" + d.IdentifierName))
```

Comment:
Really, I that practically *is* a named method--it is a single statement assigned to a well-named variable.